packages/wasm-sdk/src/dpp.rs

```diff
@@ -297,9 +297,18 @@ impl IdentityWasm {
 #[wasm_bindgen]
 pub struct DataContractWasm(DataContract);
 
-impl From<DataContract> for DataContractWasm {
-    fn from(value: DataContract) -> Self {
-        Self(value)
+impl DataContractWasm {
```

Comment:
I went through DPP methods. Actually, I think your idea to pass the platfrom version to the constructor was the best option we have. But all constructors should always require platform version then. We shouldn't create with default or first version in any case.  I'm sorry, that I wasn't clear enough in my first comment, so you reverted half correct code.

Reply:
No there actually is a trait FromWithVersion that should be used instead